### PR TITLE
US1 render the dummy array data

### PR DESCRIPTION
**Pull request by WalterRestifo**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "presets": ["next/babel"],
+  "plugins": [
+    [
+      "babel-plugin-styled-components",
+      { "ssr": true, "displayName": true, "preprocess": false }
+    ]
+  ]
```
Comment (julialoeschel) [inline .babelrc:8]: Hey Walter, what did you do here? Do you need that file?

File: components/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import styles from "../styles/Home.module.css";
+
+export default function Card({ name }) {
+  return <p className={styles.card}>{name}</p>;
```
Comment (julialoeschel) [inline components/Card.tsx:4]: We are using Styled Components, please don´t start off without.

File: components/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import Image from "next/image";
+import styles from "../styles/Home.module.css";
+import styled from "styled-components";
+import React from "react";
+
+export default function Footer() {
+  return (
+    <StyledFooter className={styles.footer}>
```
Comment (julialoeschel) [inline components/Footer.tsx:8]:
```suggestion
    <StyledFooter >
```

Comment (julialoeschel) [reply components/Footer.tsx:8]: Please dont mix styled components and regular CSS. This is bad code style.

File: pages/index.tsx [hunk 4]
```diff
@@ -1,71 +1,51 @@
...
-            className={styles.card}
-          >
-            <h2>Deploy &rarr;</h2>
-            <p>
-              Instantly deploy your Next.js site to a public URL with Vercel.
-            </p>
-          </a>
+        <div className={styles.cardContainer}>
+          {dummyArray.map(({ name }) => {
+            return <Card key={name} name={name} />;
```
Comment (julialoeschel) [inline pages/index.tsx:45]: If you can use your own data, please provide an id. You will need it for the key (more robust) and also for anything you do in future with the data.

File: pages/index.tsx [hunk 5]
```diff
@@ -1,71 +1,51 @@
...
-import styles from '../styles/Home.module.css'
+import Head from "next/head";
+import Header from "../components/Header";
+import Footer from "../components/Footer";
+import Card from "../components/Card";
+
+import styles from "../styles/Home.module.css";
+import styled from "styled-components";
+
+const dummyArray = [
```
Comment (julialoeschel) [inline pages/index.tsx:9]: Please don't call it dummy Array :) 
You will need this Array, even when you change the Data soon, and I feel better when you call it `players`or something like that.

**Review by julialoeschel — CHANGES_REQUESTED** [review]
Hey Walter. 
good work! 

Please provide a description next time for your Pull request, to let people know what they are reviewing.
And I am also curious about your app...please provide a deployment on vercel.

Please dont mix your CSS styles. We are using Styled Components, please clean your CSS before continuing

:octocat:

File: .babelrc (hunk 1 above)
Comment (WalterRestifo) [reply .babelrc:8]: Alex said it is needed. I didn´t  have the time yet  to test it, that is why I just trust him

File: components/Card.tsx (hunk 2 above)
Comment (WalterRestifo) [reply components/Card.tsx:4]: you are right, I just used the default of create-next-app and added my styled components 😅

**Review by ufelgen — COMMENTED** [review]
I have nothing to add to Julia's comments :)

**Review by MerleMak — APPROVED** [review]
Looks good! 
Keep it up 😊

**Pull request merged**